RevenueCatUI/CustomerCenter/ViewModels/ManageSubscriptions/ManageSubscriptionsViewModel.swift

```diff
@@ -264,8 +264,8 @@ private extension Array<CustomerCenterConfigData.HelpPath> {
         }
 
         return filter {
-            // if it's cancel, it cannot be a lifetime subscription
-            ($0.type != .cancel || !purchaseInformation.isLifetime) &&
+            // if it's cancel, it cannot be a lifetime subscription nor within trial period
+            ($0.type != .cancel || (!purchaseInformation.isLifetime || !purchaseInformation.isTrial)) &&
```

Comment:
I don't remember but are we filtering for purchases from other stores as well? As in, purchases from Google for example.